docs/source/developers/continuous_integration/faqs.rst

```diff
@@ -0,0 +1,45 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+FAQs
+====
+
+Q: What scripts are run when I make changes to an open PR?
```

Comment:
@AlenkaF @jonkeane @amol- could you help us to accumulate a number of questions we can make answers to? 

You may remember the not too obvious parts regarding our CI system.

Reply:
Happy to do so! Will get to it later today or first thing in the morning, if that's ok.